curation_concerns-models/app/services/curation_concerns/derivative_path.rb

```diff
@@ -0,0 +1,23 @@
+module CurationConcerns
+  class DerivativePath
+    # Path on file system where derivative file is stored
+    def self.derivative_path_for_reference(object, destination_name)
+      destination_name = destination_name.gsub(/^original_file_/, '')
+      _derivative_path(object, _extension_for(destination_name), destination_name)
+    end
+
+    def self._derivative_path(object, extension, destination_name)
+      file_name = destination_name + extension
+      File.join(CurationConcerns.configuration.derivatives_path, object.id, file_name)
```

Comment:
I suspect this should change to use `config` instead of `configuration` since the PR consolidating configs was merged.